docs/en/ingest-management/elastic-agent/otel-agent.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[otel-agent]]
+= Run {agent} as an OTel Collector
+
+preview::[]
+
+The link:https://opentelemetry.io/docs/collector/[OpenTelemetry Collector] is a vendor-neutral way to receive, process, and export telemetry data. {agent} includes an embedded OTel Collector, enabling you to instrument your applications and infrastructure once, and send data to multiple vendors and backends. 
+
+When you run {agent} in `otel` mode it supports the standard OTel Collector configuration format that defines a set of receivers, processors, exporters, and connectors. Logs, metrics, and traces can be ingested using OpenTelemetry data formats.
+
+The following steps will help you to get set up quickly using {agent} as an OTel Collector. Note that all steps are run with `sudo` as it's generally recommended to run {agent} with administrative privileges.
```

Comment:
Running with `sudo` is no longer required with OTel mode and is only needed if you're collecting from a data source that requires elevated privileges!

Reply:
I've removed this note about needing `sudo` and changed all examples to not have the `sudo` prefix.